docs/_articles/en/einstein/einstein-example-prompts.md

```diff
@@ -29,25 +29,13 @@ Currently the CodeGen2.5 model can pick up on the following context from your pr
 
 We’re listing some tips for prompt writing and also providing you with some examples that help you craft your prompts. An effective prompt consists of these four parts:
 
-1. **The Action**: The task you want the AI to accomplish.
+**Action**: The task you want the AI to accomplish. For example, "_Create an Apex batch class named "OpenPositionMatcher"_".
 
-For example:
```

Comment:
FWIW, I prefer the old way you had this.  You can indent the "For example" stuff (so it looks better as part of the numbered step) by simply pre-pending 4 spaces.  Just my $.02.